src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    private String name;
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
```

Comment:
```suggestion
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
```

why do you need this? you don't use name

Reply:
Okay, I will remove it